src/App.test.js

```diff
@@ -1,16 +1,46 @@
-import { render, screen } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 import App from './App';
 import React from 'react'
+import * as fetchJson from "./fetchJson";
+import AstronautsTable from './AstronautsTable';
 
 
-test('renders learn react link', () => {
+test('renders loading screen', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  const loadingText = screen.getByText(/Loading/i);
+  expect(loadingText).toBeInTheDocument();
 });
 
-test('renders test message', () => {
+test('renders graphic', () => {
   render(<App />);
-  const linkElement = screen.getByText(/Grace's app/i);
-  expect(linkElement).toBeInTheDocument();
+  const isGraphicThere = document.getElementsByClassName("Astronaut");
+  expect(isGraphicThere.length).toBe(1);
+});
+
+test('renders graphic', () => {
```

Comment:
Test name doesn't match test

Reply:
🤦‍♀️ Thank you, good catch. I have got to get better about  remembering to rename these copy-pasted tests.